projects/ion/src/lib/button/button.component.spec.ts

```diff
@@ -222,6 +222,26 @@ describe('ButtonComponent with dropdown', () => {
     });
   });
 
+  it('should close the dropdown when the outside is clicked "', async () => {
+    const options = [{ label: 'Option 1' }, { label: 'Option 2' }];
+
+    const button = await sut({
+      label: defaultName,
+      options,
+    });
+
+    fireEvent.click(button);
+
+    const fakeDiv = document.createElement('div');
+    fakeDiv.setAttribute('data-testid', 'fake-div');
+    document.body.appendChild(fakeDiv);
+
+    fireEvent.click(fakeDiv);
```

Comment:
Why double click?

Reply:
The test is not recognizing the first

Reply:
Probably because the test above is letting the dropdown opened and Jest is memorizing this states. 😄

Reply:
I'm not sure, but I believe you can click directly on the document.body. So you don't need to create a fake div